internals/cli/cmd_run.go

```diff
@@ -59,7 +59,7 @@ type sharedRunEnterOpts struct {
 var sharedRunEnterArgsHelp = map[string]string{
 	"--create-dirs": "Create {{.DisplayName}} directory on startup if it doesn't exist",
 	"--hold":        "Do not start default services automatically",
-	"--http":        `Start HTTP API listening on this address (e.g., ":4000") and expose open-access endpoints`,
+	"--http":        `Start HTTP API listening on this address in "[ipv4_or_v6_address]:port" format (for example, ":4000")`,
```

Comment:
Square brackets are a bit confusing, since they are used for v6 addresses.
```suggestion
	"--http":        `Start HTTP API listening on this address in "<address>:port" format (for example, ":4000", "192.0.2.0:4000", "[2001:db8::1]:4000")`,
```

Same for `docs/reference/cli-commands.md`.

Reply:
Thanks, good call - done